kernel/build/config-arm64

```diff
@@ -1122,7 +1122,7 @@ CONFIG_PCP_BATCH_SCALE_MAX=5
 CONFIG_PHYS_ADDR_T_64BIT=y
 CONFIG_MMU_NOTIFIER=y
 CONFIG_KSM=y
-CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
+CONFIG_DEFAULT_MMAP_MIN_ADDR=65536
```

Comment:
this seems like a bug in the checker script: https://cateee.net/lkddb/web-lkddb/DEFAULT_MMAP_MIN_ADDR.html

> On arm and other archs it should not be higher than 32768

Reply:
ah... `arm64/arm/` :)